app/(drawer)/(tabs)/send.tsx

```diff
@@ -28,7 +28,7 @@ const sendPillContent = () => {
 };
 
 export default function Send() {
-  const { user, tokens, privy } = useAppData();
+  const { user, tokens, privy, updateUser } = useAppData();
```

Comment:
`updateUser` is not used anywhere
```suggestion
  const { user, tokens, privy } = useAppData();
```